ui/src/app/pages/analysis/leonardo-app-launcher.tsx

```diff
@@ -53,6 +66,16 @@ export const progressStrings: Map<Progress, string> = new Map([
   [Progress.Redirecting, 'Redirecting to the notebook server'],
 ]);
 
+export const terminalProgressStrings: Map<Progress, string> = new Map([
+  [Progress.Unknown, 'Connecting to the terminal server'],
+  [Progress.Initializing, 'Initializing terminal server, may take up to 5 minutes'],
+  [Progress.Resuming, 'Resuming terminal server, may take up to 1 minute'],
+  [Progress.Authenticating, 'Authenticating with the terminal server'],
+  [Progress.Copying, 'Copying notebooks onto the server'],
+  [Progress.Creating, 'Creating the new terminal'],
+  [Progress.Redirecting, 'Redirecting to the terminal server'],
```

Comment:
I don't think I would call it "terminal server". I think I would either use "notebook server" , or "terminal" throughout